src/components/BuildBaronAndAnnotations/BuildBaron.tsx

```diff
@@ -34,6 +34,7 @@ const BuildBaron: React.FC<Props> = ({
     {bbData && (
       <BuildBaronContent
         bbData={bbData.buildBaron}
+        annotationId={annotation?.id}
```

Comment:
Is `annotation` nullable? If so what happens if its undefined?
The props for BuildBaronContent apply that `annotationId` is required

Reply:
It is used by the moveAnnotation mutation, and that button only shows up in the table row of issues or suspected issues. So it wouldn't surface unless there is an issue or a suspected issue.